File: my-server-spring/src/main/java/com/example/t3/investInfo/model/InvestInfo.java
package com.example.t3.investInfo.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class InvestInfo {
//	  public i_id : number = 0;
//	  public u_id : string;
//	  public d_pageno : number;
//	  public i_investmoney : number;
	
	private int i_id;
	private String u_id;
	private int d_pageno;
	private int i_investmoney;


}

File: my-client-angular/src/app/joonja/postarea/purchase/InvestInfo.ts
export class InvestInfo {
//   `i_id`           INT            NOT NULL AUTO_INCREMENT    COMMENT '순번',
// `u_id`           VARCHAR(50)    NOT NULL                   COMMENT '유저아이디',
// `d_pageno`       INT            NOT NULL                   COMMENT '페이지번호',
// `i_investmoney`  INT			   NOT NULL DEFAULT 0		   COMMENT '투자금액',
//
  public i_id : number = 0;
  public u_id : string;
  public d_pageno : number;
  public i_investmoney : number;
}
